config.js

```diff
@@ -51,6 +51,23 @@ config.server = {
     maxPageSize: '100'
 };
 
+// Cors Configuration
+config.cors = {
```

Comment:
I understand this configuration doesn't need to be mapped to equivalent env vars as many other do (see doc/manuals/running.md file). However, I think it would be a good idea to include a small section at the end of that file (doc/manuals/running.md) explaining how to configure/use CORS.